UtopiaCity/Controllers/Sport/RequestToAdministrationController.cs

```diff
@@ -0,0 +1,188 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using UtopiaCity.Models.Sport;
+using UtopiaCity.Services.Sport;
+using UtopiaCity.ViewModels.Sport;
+
+namespace UtopiaCity.Controllers.Sport
+{
+    public class RequestToAdministrationController : Controller
+    {
+        private readonly RequestToAdministrationService _requestToAdministrationService;
+        private readonly SportComplexService _sportComplexService;
+        private readonly IMapper _mapper;
+
+        public RequestToAdministrationController
+            (RequestToAdministrationService requestToAdministrationService, SportComplexService sportComplexService, IMapper mapper)
+        {
+            _requestToAdministrationService = requestToAdministrationService;
+            _sportComplexService = sportComplexService;
+            _mapper = mapper;
+        }
+
+        public IActionResult AllRequestsToAdministration()
+        {
+            List<RequestToAdministration> allRequests = _requestToAdministrationService.GetAllRequestsToAdministration();
+            List<RequestToAdministrationViewModel> viewModels = _requestToAdministrationService.CreatingRequestToAdministationViewModel(allRequests, _mapper);
+            ViewBag.SportComplexesIds = _sportComplexService.GetAllSportComplexesIds();
+            ViewBag.IsMainPage = true;
+            return View(viewModels);
+        }
+
+        public IActionResult AllRequestsToAdministrationByDate(DateTime date)
+        {
+            List<RequestToAdministration> allRequests = _requestToAdministrationService.GetRequestsToAdministrationByDate(date);
```

Comment:
probably can be `var` - it is more or less clear what type it is, also temporary variable